evap/evaluation/tools.py

```diff
@@ -275,7 +275,10 @@ def send_publish_notifications(courses):
             publish_notifications[course.responsible_contributor].add(course)
 
     for user, course_set in publish_notifications.items():
-        EmailTemplate.send_publish_notifications_to_user(user, list(course_set))
+        if not template:
+            template = EmailTemplate.objects.get(name=EmailTemplate.PUBLISHING_NOTICE)
```

Comment:
It would be 3 places where this line would be put into the method call. I would prefer to keep it that way as executing this method with a specific template != PUBLISHING_NOTICE seems to be the special case.